### DISCUSSION TITLE
Nested relationship reference for labelField

### DISCUSSION THREAD

**Discussion by jschuur** [keystonejs/keystone#8185] [Feature requests]
I would like to be able to reference the field of a relationship and make that the `labelField` for a parent list. Something like this:

```JS
{
  fields: {
    campaignBuild: relationship({
      ref: 'CampaignBuild.adNetworkReportJob',
      many: false,
    }),
    // ...
  },
  ui: {
     labelField: 'campaignBuild.name',
  }
}
```

Better yet, leaving `.name` off will default to the label field of the relationship.

At this point, the workaround is to just create a virtual field and do that lookup yourself, but this looks like a standard enough use case to offer a convenience shorthand for.

This would only make sense for `many: false` relationships of course.

**Comment by jj-matos** [comment]
Same issue here. Any solution?
